File: culkay.py
t = int(input())
for _ in range(t):
    n = int(input())
    a = [0]*n
    for j in range(0, n, 1):
        a[j] = int(input())
    hgh = 0
    low = 0
    for i in range(0, n-1, 1):
        if a[i+1]>a[i]:
            hgh+=1
        elif a[i+1]<a[i]:
            low += 1
    print(hgh, low)

File: culkay.cpp
#include <bits/stdc++.h>
using namespace std;
#define FAST_IO ios::sync_with_stdio(false); cin.tie(nullptr);
#define nl '\n'
#define int long long
void solve() {
    //My solution logic here
    int n;
    cin>>n;
    vector<int>a(n);
    for(int i = 0; i<n; i++) cin>>a[i];
    int hgh = 0, low = 0;
    for(int i = 0; i<n-1; i++){
        if(a[i+1]>a[i]) hgh++;
        else if(a[i+1]<a[i]) low++;
    }
    cout<<hgh<<" "<<low<<nl;
}

signed main(){
    int t;
    //t = 1;
    cin >> t; 
    while (t--) {
        solve();
    }

    return 0;
}
